tests/Doctrine/Migrations/Tests/InlineParameterFormatterTest.php

```
@@ -54,11 +58,13 @@ public function testFormatParameters(): void
             'unknown',
             'unknown',
             'unknown',
+            ParameterType::STRING,
+            ArrayParameterType::INTEGER,
```

```suggestion
            class_exists(ArrayParameterType::class) ? ArrayParameterType::INTEGER : Connection::PARAM_INT_ARRAY,
```

This way, we can avoid to bump dependencies in a bugfix release.

---

pushed, and this is what I was worried about - phpstan failure... :O do you want to me to add ignore on line?

---

Yes, we can ignore the error.

---

@derrabus pushed, green now.

any chance to get it released soon? thanks